File: java/Harddrive.java
class Harddrive extends Storage implements PerangkatPenyimpanan {
  private String tipeDrive;

  public Harddrive(int kapasitasGB, String tipeDrive, String merk, String nama) {
      super(kapasitasGB, merk, nama);
      this.tipeDrive = tipeDrive;
  }

  @Override
  public void info() {
      System.out.println("Harddrive: " + tipeDrive + " " + merk + " " + nama + " (" + kapasitasGB + " GB)");
  }

  @Override
  public void kapasitasMaksimal() {
      System.out.println("Harddrive ini mendukung hingga 4TB penyimpanan.");
  }
}

File: python/harddrive.py
from storage import Storage
from perangkat_penyimpanan import PerangkatPenyimpanan

class Harddrive(Storage, PerangkatPenyimpanan):
    def __init__(self, kapasitas_gb: int, tipe_drive: str, merk: str, nama: str):
        super().__init__(kapasitas_gb, merk, nama)
        self.tipe_drive = tipe_drive

    def info(self):
        print(f"Harddrive: {self.tipe_drive} {self.merk} {self.nama} ({self.kapasitas_gb} GB)")

    def kapasitas_maksimal(self):
        print("Harddrive ini mendukung hingga 4TB penyimpanan.")
